packages/jokul/src/components/menu/MenuItem.tsx

```diff
@@ -67,13 +66,11 @@ export const MenuItem = forwardRef(function MenuItem<
                 {children}
                 {external && (
                     <div className={"jkl-menu-item__arrow"}>
-                        <ArrowNorthEastIcon />
+                        <Icon>{"\ue89e"}</Icon>
```

Comment:
Nå skjønte jeg konteksten for spørsmål om unicode points :)

Her kunne du bare brukt https://github.com/fremtind/jokul/blob/main/packages/jokul/src/components/icon/icons/OpenInNewIcon.tsx